plugins/labhub.py

```diff
@@ -387,3 +388,65 @@ def pr_stats(self, msg, match):
                     state=type(self).community_state(pr_count)
                  )
         yield reply
+
+    @re_botcmd(pattern=r'pr\s+list\s+([^/]+)',  # Ignore LineLengthBear, PyCodeStyleBear
+               re_cmd_name_help='pr list <repo>',
+               flags=re.IGNORECASE)
+    def pr_list(self, msg, match):
+        """List PRs ready to be merged."""  # Ignore QuotesBear
+        repo_name = match.groups(1)[0]
+
+        try:
+            mrs = self.REPOS[repo_name].merge_requests()
+        except KeyError:
+            return 'Repository doesn\'t exist.'
+        else:
```

Comment:
this `else:` causes an unnecessary indent. (yes, I know you will have seen it elsewhere ... but that is wrong)

there is a return in the `except:` block above.

And any other exception will cascade up.

And `try: else:` is only useful when one of the exception paths does not exit the function. (and some strange cases including use of `finally:`.  See https://stackoverflow.com/questions/16138232/is-it-a-good-practice-to-use-try-except-else-in-python

Reply:
Ok, got you, I see the difference now